app/electron/main.js

```diff
@@ -0,0 +1,38 @@
+const { app, BrowserWindow } = require('electron');
+const { spawn, execSync } = require("child_process");
+const path = require('path');
+const url = require('url');
+let mainWindow;
+
+const isDev = process.mainModule.filename.indexOf('app.asar') === -1 ||
+    process.mainModule.filename.indexOf('app') === -1;
+
+function createWindow () {
+  let serverFilePath;
+  if(!isDev) {
+     serverFilePath = path.join(process.resourcesPath,'./electron/server');
+     spawn(serverFilePath)
```

Comment:
Add a `;` at the end of this line. Actually I think the linter needs to be pointed to `app/` now.